test/fixtures/output/asyncapi/import-with-overwrite.golden

```diff
@@ -1,2 +1,4 @@
-Schema registered under subject topic1-value with ID 1.
+Schema registered under subject "topic1-value" with ID "1".
+Tags added to schema "1".
+Tags added to Kafka topic "topic1"
 AsyncAPI specification imported from "./test/fixtures/input/asyncapi/asyncapi-spec.yaml"
```

Comment:
Let's leave this out. As long as the above messages are printed and the command exits without and error, the user can assume that the specification has been imported.